plugins/rocks.xml.webhelp/xsl/topic.xsl

```diff
@@ -117,10 +117,11 @@
           <div class="breadcrumb-container max-width">
                 <xsl:call-template name="generate-custom-breadcrumbs"/>
             </div>
-          
+
             <div class="top-nav-buttons-container max-width">
-                <a class="prev-button" href="#">&#8592; PREV</a>
-                <a class="next-button" href="#">NEXT &#8594;</a>
+                <!-- TODO: Replace with working prev/next buttons -->
+                <a class="prev-button" href="#"><pre class="button-prev-nav"/> PREV</a>
```

Comment:
We should use default dita-ot 'getVariable' template here and use translated strings.

Reply:
OK, will do